Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,26 +1,59 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
+        public List<Item> Items { get; set; }
+
+        public Inventory()
         {
-            return new Item[0];
+            Items = new List<Item>();
         }
 
-        public Inventory()
+        /// <summary>
+        /// Returns the item it found based on the name you gave, if no item is found returns null
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public Item Contains(string name)
         {
-            _items = new Item[0];
+            foreach (Item item in Items)
+            {
+                if (item.Name == name)
+                {
+                    return item;
+                }
+            }
+            return null;
         }
 
         public Item[] GetItems(string name)
```

Comment:
The main goal of this homework is to refactor out the arrays and replace with Lists. Will need to change the return type and amend the code inside :)